tark/tark_web/urls.py

```diff
@@ -136,6 +136,20 @@
         )
     ),
 
+    # new mane list page
+    url(
+        r'^mane_list_new/$',
```

Comment:
I think it would be better to make the changes directly on `mane_list.html` (if they're meant to display the same content) instead of creating a new file `mane_list_new.html`, this will allow us to see what changes are made, except if there was a reason for having them both (I mean `mane_list.html` and `mane_list_new.html`)

Reply:
Hi Bilal thanks a lot for your comments.
There is a slight difference between mane_list and mane_list_new.html. mane_list.html is required for the API https://tark.ensembl.org/api/#!/transcript/transcript_manelist_list

Reply:
Hey @ranjit-s 

Do we need mane_list page after this new mane_list_new page is implemented? 

I don't see mane_list page linked from anywhere else on the website. If mane_list_new is supposed to replace mane_list, I suggest you retain the old URL.

Reply:
Ok @sanjay-boddu , @bilalebi , thanks a lot for the comments. I will make changes to mane_list.html, remove the new file and resubmit the PR.